apps/web-app/pages/join/profile.vue

```diff
@@ -94,7 +94,6 @@
         '5xl'
       ) as string;
     }
-
```

Comment:
we should also do the same thing here, after upload set the image url to the returned avatarId